dandi/files.py

```diff
@@ -1024,14 +1026,16 @@ def mkzarr() -> str:
                 pluralize(len(old_zarr_files), "file"),
             )
             a.rmfiles(old_zarr_files, reingest=False)
-        lgr.debug("%s: Waiting for server to calculate Zarr checksum", asset_path)
-        yield {"status": "server calculating checksum"}
-        client.post(f"/zarr/{zarr_id}/ingest/")
-        while True:
-            sleep(2)
-            r = client.get(f"/zarr/{zarr_id}/")
-            if r["status"] == "Complete":
-                break
+            need_ingest = True
+        if need_ingest:
+            lgr.debug("%s: Waiting for server to calculate Zarr checksum", asset_path)
+            yield {"status": "server calculating checksum"}
+            client.post(f"/zarr/{zarr_id}/ingest/")
+            while True:
+                sleep(2)
+                r = client.get(f"/zarr/{zarr_id}/")
+                if r["status"] == "Complete":
+                    break
         lgr.info("%s: Asset successfully uploaded", asset_path)
```

Comment:
well, if we have not uploaded anything, may be we should not state that here either and move this under `if need_ingest` while in `else:` do state that no updates to the Asset were made

Reply:
Done: 95cedc9.